Wrong event_type value passed in to show_help when getting xrc async events

diff -r e4bab4451664 ompi/mca/btl/openib/btl_openib_async.c
--- a/ompi/mca/btl/openib/btl_openib_async.c	Tue May 25 01:30:35 2010 +0200
+++ b/ompi/mca/btl/openib/btl_openib_async.c	Thu Jul 15 14:09:33 2010 +0200
@@ -350,13 +350,15 @@ static int btl_openib_async_deviceh(stru
             case IBV_EVENT_SRQ_ERR:
                 orte_show_help("help-mpi-btl-openib.txt", "of error event",
                     true,orte_process_info.nodename, orte_process_info.pid,
-                    event.event_type, openib_event_to_str(event.event_type),
+                    event_type,
+                    openib_event_to_str((enum ibv_event_type)event_type),
                     xrc_event ? "true" : "false");
                 break;
             case IBV_EVENT_PORT_ERR:
                 orte_show_help("help-mpi-btl-openib.txt", "of error event",
                     true,orte_process_info.nodename, orte_process_info.pid,
-                    event.event_type, openib_event_to_str(event.event_type),
+                    event_type,
+                    openib_event_to_str((enum ibv_event_type)event_type),
                     xrc_event ? "true" : "false");
                 /* Set the flag to indicate port error */
                 device->got_port_event = true;
@@ -385,7 +387,7 @@ static int btl_openib_async_deviceh(stru
             default:
                 orte_show_help("help-mpi-btl-openib.txt", "of unknown event",
                         true,orte_process_info.nodename, orte_process_info.pid,
-                        event.event_type, xrc_event ? "true" : "false");
+                        event_type, xrc_event ? "true" : "false");
         }
         ibv_ack_async_event(&event);
     } else {

